<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetTCPAssistantServer</class>
 <widget class="QWidget" name="WidgetTCPAssistantServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络调试助手-服务端  (Ithaca weili_yzzcq@163.com)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="6,1,1,1,1,0">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="10">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="7" colspan="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>48</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="11">
    <widget class="QTextEdit" name="textEdit_recv"/>
   </item>
   <item row="1" column="8">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>端口：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>58</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>通信协议：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QPushButton" name="btn_listen">
     <property name="text">
      <string>开始监听</string>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QPushButton" name="btn_disconnect">
     <property name="text">
      <string>断开连接</string>
     </property>
    </widget>
   </item>
   <item row="3" column="10">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="11">
    <widget class="QTextEdit" name="textEdit_send"/>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="comboBox_protocl">
     <item>
      <property name="text">
       <string>TCP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UDP</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="6" colspan="2">
    <widget class="QComboBox" name="comboBox_IP"/>
   </item>
   <item row="3" column="4" colspan="2">
    <widget class="QPushButton" name="btn_stopListen">
     <property name="text">
      <string>停止监听</string>
     </property>
    </widget>
   </item>
   <item row="3" column="9">
    <widget class="QPushButton" name="btn_send">
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>服务端IP:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <widget class="QLineEdit" name="lineEdit_port">
     <property name="text">
      <string>8888</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="10">
    <spacer name="horizontalSpacer_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" colspan="9">
    <widget class="ComboBoxRefresh" name="comboBox_clients"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBoxRefresh</class>
   <extends>QComboBox</extends>
   <header>ComboBoxRefresh.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
